processor/src/main/java/com/linecorp/decaton/processor/runtime/Processors.java

```diff
@@ -33,15 +34,18 @@
 
     private final List<DecatonProcessorSupplier<T>> suppliers;
     private final DecatonProcessorSupplier<byte[]> retryProcessorSupplier;
+    private final Deserializer<String> keyDeserializer;
```

Comment:
Mentioned in the above comment as well but I think this should be a property of `ProcessorSubscription`.